doc/manual/source/interaction/geoserver_web-api.rst

```diff
@@ -201,61 +183,51 @@ as a JSON object.
-JSON formatted response by appending ``.json`` to the PUT URL.
+Again, the ``Content-Type`` must be set correctly for the plugin to parse the connection parameters and you may request a JSON-formatted response by appending ``.json`` to the PUT URL.
 
-Creating a GeoGig DataStore with REST
+Creating a GeoGig datastore with REST
 -------------------------------------
 
-To create a GeoGig DataStore through the GeoServer REST API, you must already have a GeoGig Repository configured in GeoServer.
-See :ref:`geoserver_web-api-plugin-init` for creating a GeoGig Repository in GeoServer.
-
-Once you have a GeoGig repository, you just need to follow the
-`standard procedure <http://docs.geoserver.org/stable/en/user/rest/api/datastores.html>`_, 
-knowing which data store connection parameters to use.
+To create a GeoGig datastore through the GeoServer REST API, you must already have a GeoGig repository configured in GeoServer. See :ref:`geoserver_web-api-plugin-init` for creating a GeoGig Repository in GeoServer.
 
-That is, issuing a ``POST`` request to ``/workspaces/<ws>/datastores[.<format>]``, where the request body for the XML
-representation is like:
+Once you have a GeoGig repository, you need only follow the `standard procedure <http://docs.geoserver.org/stable/en/user/rest/api/datastores.html>`_, knowing which datastore connection parameters to use.
 
-::
+That is, issuing a ``POST`` request to ``/workspaces/<ws>/datastores[.<format>]``, where the request body for the XML representation appears::
 
    <dataStore>
-      <name>${data store name}</name>
+      <name>${datastore name}</name>
       <connectionParameters>
          <entry key="geogig_repository">${repository URI}</entry>
          <entry key="branch">${branch}</entry>
       </connectionParameters>
    </dataStore>
 
-That's all the information needed to create a GeoGig data store.
+That's all the information needed to create a GeoGig datastore.
 
-* ${data store name} is the name to be given to the data store, which then will be accessible through ``/workspaces/<ws>/datastores/<data store name>``. 
+* ${datastore name} is the name to be given to the datastore, which then will be accessible through ``/workspaces/<ws>/datastores/<datastore name>``.
 * ${repository URI} is a GeoServer URI string that identifies the GeoGig repository in GeoServer. It should be in the form ``geoserver://<repository name>``
-* ${branch} is optional, and represents the name of the branch the data store is going to serve its data from. If not given, defaults to using whatever branch is the currently checked out one in the repository whenever the data store is used.
+* ${branch} is optional and represents the name of the branch the datastore is going to serve its data from. If not given, defaults to using whichever is the currently checked out branch in the repository while the datastore is used.
```

Comment:
The wording for the default here is not correct (the original isn't worded well either). It should be:

`If not provided, the datastore will use whichever is the currently checked out branch in the repository each time it (or the datastore) is accessed.`

Meaning, you access the datastore *now*, you are operating against the branch currently checked out. If the checked out branch changes, and you later access the datastore again, you will be operating against whichever branch is checked out at the time you access the datastore.